test/LaunchDarkly.EventSource.Tests/EventParserTest.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace LaunchDarkly.EventSource.Tests
 {
-    public class EventParserTests
+    public class EventParserTest
```

Comment:
Trivial, but our usual naming style is to call a test class `___Test` instead of `___Tests`, since conceptually Xunit instantiates this class once for each test.